components/generator/imagevector/build.gradle.kts

```diff
@@ -9,6 +9,7 @@ dependencies {
     implementation(libs.kotlinpoet)
 
     testImplementation(projects.components.parser)
+    testImplementation(projects.components.extensions.dependencyProject.sourceSets.test.get().output)
```

Comment:
I believe you can move `ResourceUtils` into main source set instead of the test, then we can `testImplementation` that project here.

Reply:
yes, can be